core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -392,6 +392,30 @@ private class LiveExecutorStageSummary(
 
 }
 
+private class LiveSpeculationStageSummary(
+    stageId: Int,
+    attemptId: Int) extends LiveEntity {
+
+  var numTasks = 0
+  var numActiveTasks = 0
+  var numCompletedTasks = 0
+  var numFailedTasks = 0
+  var numKilledTasks = 0
+  var killedTasksSummary : Map[String, Int] = Map()
```

Comment:
It's nit but how about `killedSummary` like other `Live*` classes.

Reply:
As we have agreed to remove the `killedTasksSummary` for speculation specifically, this comment is not relevant anymore.